<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(128, 159, 168);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>220</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>23</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Agendamento</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>1330</width>
      <height>510</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>110</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>ID </string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>STATUS</string>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nº CADASTRO</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>NOME PET</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>DATA</string>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>HORA</string>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>BANHO</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TOSA</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TAXI PET</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ID TOSA</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TIPO TOSA</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>OBSERVAÇÃO</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>CONTATO DONO</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>IDADE</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>PORTE</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>SEXO</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>CASTRADO</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>RAÇA</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>610</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>VISUALIZAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>610</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>FINALIZAR</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
